source/set_up_3dsecure/index.html.md.erb

```diff
@@ -36,9 +38,10 @@ Worldpay does not apply any SCA exemptions to transactions.
 1. In the GOV.UK Pay admin tool, select the account you want to set up on the __My services__ page.
 1. In __Settings__, make sure 3D Secure is set to __On__.
 1. In __Your PSP__, enter your 3DS Flex credentials.
+1. Select __Save credentials__.
```

Comment:
Whoops, should have updated this myself when I tinkered with this page!

Reply:
I missed it too, to be fair.